src/app/zap-templates/templates/app/callback-stub-src.zapt

```diff
@@ -132,3 +132,10 @@ void __attribute__((weak)) MatterPostAttributeChangeCallback(
     uint8_t type, uint16_t size, uint8_t * value)
 {
 }
+
+void __attribute__((weak)) MatterPreExecuteCommandCallback(chip::app::CommandHandler * commandObj,
+                                                           const chip::app::ConcreteCommandPath & commandPath)
+{
+    (void)commandObj;
+    (void)commandPath;
+}
```

Comment:
It's really too bad we have all these global callbacks as the public API with no context whatsoever passed in. It really keeps baking-in more and more that we cannot cleanly have multiple instances of Matter cluster handling in 1 process, or even be able to initialize clusters with particular context without going through globals/singletons everywhere.

Won't block this PR since it's based on the existing model that is already unfortunate, but this just highlights it more.